app/jobs.js

```diff
@@ -71,6 +66,14 @@ var job = {
             }
             catch (ex) {
                 await transaction.rollback();
+                if (ex.name=="SequelizeUniqueConstraintError"){
+                    logger.log({
+                        level: 'info',
+                        message: 'report already exist',
+                      });
+                    var response = { success: 0, message: "report with this visulaizationid already exit" }
+                    return response;
```

Comment:
it can be return we do not need to create one more var

return { success: 0, message: "report with this visulaizationid already exit" };